packages/blockchain-wallet-v4/src/redux/kvStore/lockbox/selectors.js

```diff
@@ -21,7 +21,7 @@ export const getMetadata = path([kvStorePath, LOCKBOX])
 
 export const getDevices = state =>
   getMetadata(state)
-    .map(path(['value', 'devices']))
+    .map(pathOr([], ['value', 'devices']))
```

Comment:
This change may be unnecessary because of https://github.com/blockchain/blockchain-wallet-v4-frontend/pull/1316/files#diff-71a2ae1bf9b609fdbabff840d9d9f25eL38 fix. Need to recheck